lib/src/wizard/Wizard.tsx

```diff
@@ -146,20 +147,30 @@ const StepsContainer = styled.div`
     props.margin && typeof props.margin === "object" && props.margin.left ? spaces[props.margin.left] : ""};
 `;
 
-const StepContainer = styled.div`
+type StepContainerProps = {
+  mode: Mode;
+  lastStep: boolean;
+};
+const StepContainer = styled.div<StepContainerProps>`
   display: inline-flex;
-  ${(props) => (props.mode === "vertical" ? "" : "align-items: center;")}
+  ${(props) => props.mode !== "vertical" && "align-items: center;"}
   flex-grow: ${(props) => (props.lastStep ? "0" : "1")};
   flex-direction: ${(props) => (props.mode === "vertical" ? "column" : "row")};
-  ${(props) => (props.mode === "vertical" ? "width: 100%;" : "")}
+  ${(props) => props.mode === "vertical" && "width: fit-content;"}
 `;
 
-const Step = styled.button`
-  border: none;
-  background: inherit;
+type StepProps = {
```

Comment:
Missing `disabled` prop.

Reply:
The step is a button tag so it has a [`disabled`](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/disabled) prop by default from the HTML implementation, it's not necessary to add it again.